lib/new_relic/agent/transaction.rb

```diff
@@ -816,13 +816,9 @@ def record_queue_time
       end
 
       def had_error_affecting_apdex?
-        @exceptions.each do |exception, options|
-          ignored = NewRelic::Agent.instance.error_collector.error_is_ignored?(exception)
-          expected = options[:expected]
-
-          return true unless ignored || expected
+        @exceptions.each.any? do |exception, options|
```

Comment:
Do you recall the structure of `@exceptions`? From this file it looks like a Hash, but I'm not sure what's inside the hash. (Can also talk about this sync in standup)

Reply:
Hi @kaylareopelle! Yep. `@exceptions` here is a hash. It looks like this:

```ruby
{ my_error => { expected: false },
  my_other_error => { expected: true } }
```

The key is an instance of a Ruby error class and the value is an options hash related to how the agent is expected to treat that error. In this context we're only concerned with the `:expected` key inside that inner hash.

Reply:
Previously we were using an early `return true` if any key/value pair was found to satisfy the criteria, and then using `return false` if none were found. I refactored that to leverage `Enumerable#any?` instead.